src/app/pages/wallets/management/wallet-editor/pages-wallets-management-editor.component.spec.ts

```diff
@@ -1,18 +1,20 @@
 import { ComponentFixture, TestBed } from '@angular/core/testing';
-
 import { PagesWalletsManagementEditorComponent } from './pages-wallets-management-editor.component';
 import { MAT_DIALOG_DATA, MatDialogRef } from '@angular/material/dialog';
 import { ReactiveFormsModule } from '@angular/forms';
 import { MaterialModule } from '../../../../material.module';
 import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { By } from '@angular/platform-browser';
 
-describe('PagesWalletsManagementEditorComponent', () => {
+describe('WalletFormModalComponent', () => {
```

Comment:
Seems like something went wrong during merge -> this should be `'PagesWalletsManagementEditorComponent'`